### PR TITLE
add Oneplus3 support

### PR DESCRIPTION
**Pull request by SirRGB**
https://github.com/openandroidinstaller-dev/openandroidinstaller/issues/273

### CODE REVIEW

**Comment by SirRGB** [comment]
Needs https://github.com/openandroidinstaller-dev/openandroidinstaller/pull/362 merged first to avoid conflicts

File: README.md [hunk 1]
```diff
@@ -163,17 +163,18 @@ Motorola | moto z | [griffin](https://wiki.lineageos.org/devices/griffin) | | te
...
-OnePlus | 6 | [enchilada](https://wiki.lineageos.org/devices/enchilada) | | tested
-OnePlus | 6T | [fajita](https://wiki.lineageos.org/devices/fajita) | | tested
-OnePlus | 7 | [guacamoleb](https://wiki.lineageos.org/devices/guacamoleb) | | tested
-OnePlus | 7 Pro | [guacamole](https://wiki.lineageos.org/devices/guacamole) | | tested
-OnePlus | 7T | [hotdogb](https://wiki.lineageos.org/devices/hotdogb) | | tested
-OnePlus | 7T Pro | [hotdog](https://wiki.lineageos.org/devices/hotdog) | | tested
-OnePlus | Nord | [avicii](https://wiki.lineageos.org/devices/avicii) | | tested
-OnePlus | Nord N200 | [dre](https://wiki.lineageos.org/devices/dre) | | tested
-OnePlus | 9 | lemonade | | under development
+OnePlus | 3/3T | [oneplus3](https://wiki.lineageos.org/devices/oneplus3) | A3003, A3000, A3010 | untested
```
Comment (tsterbak) [inline README.md:166]: Are you certain all of these models are supported?

Comment (SirRGB) [reply README.md:166]: The A3010 seems to be the China model and isnt officiall supported? Will update soon.
There seem to be people having flashed an A3010, so Im not quite sure what to think of it.

https://wiki.lineageos.org/devices/oneplus3/variant1/
https://wiki.lineageos.org/devices/oneplus3/variant2/
https://en.m.wikipedia.org/wiki/OnePlus_3T
https://xdaforums.com/t/rom-official-weekly-lineageos-14-1-for-oneplus-3-3t.3496850/page-270#post-73436310

Comment (SirRGB) [reply README.md:166]: https://xdaforums.com/t/rom-official-nightly-lineageos-18-1-for-oneplus-3-3t.4230665/post-89178101

3010 seems to work
I redid all moto model names locally
Just have to redo op ig

**Review by MagicLike — APPROVED** [review]
LGTM

**Pull request merged**